### ISSUE TITLE
Unable to send notification: EOF

### ISSUE THREAD

**Issue by vadimDidenko** [AcalephStorage/consul-alerts#207]
I trying use consul-alerts with default template
Consul KV properties:
`
consul-alerts/config/notifiers/email/enabled  true
consul-alerts/config/notifiers/email/password {AWS SES user pass}
consul-alerts/config/notifiers/email/port 25
consul-alerts/config/notifiers/email/receivers ["{my email}"]
consul-alerts/config/notifiers/email/sender-email  {sender email}
consul-alerts/config/notifiers/email/url email-smtp.us-east-1.amazonaws.com
consul-alerts/config/notifiers/email/username {AWS SES user name}
`
I start alerts with debug logging and see error after trying send email : Unable to send notification: EOF

**Comment by FRosner** [comment]
Hi @vadimDidenko!

Please provide a [Short, Self Contained, Correct Example](http://sscce.org/) so someone can reproduce the issue. Unfortunately your KV properties are not properly formatted and hard to parse.

What is the actual behaviour and what the expected one?
